### PR TITLE
fix: cairo1-run array argument's length can not be less than 2

### PR DESCRIPTION
**Pull request by catusax**
# fix: cairo1-run array argument's length can not be less than 2

## Description

if the program has an argument which is an empty array or the length is 1, cairo1-run fails with error:

```
thread 'main' panicked at cairo1-run/src/main.rs:73:78:
called `Result::unwrap()` on an `Err` value: FromStrError
```

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [x] CHANGELOG has been updated.

### CODE REVIEW

**Review by fmoletta — APPROVED** [review]
Thanks for the fix! Please add this PR to the CHANGELOG.md file so we can record the changes!

**Comment by catusax** [comment]
> Thanks for the fix! Please add this PR to the CHANGELOG.md file so we can record the changes!

CHANGELOG has been updated. please check again

File: CHANGELOG.md [hunk 1]
```diff
@@ -152,6 +152,8 @@
 
 * feat: output builtin features for bootloader support [#1580](https://github.com/lambdaclass/cairo-vm/pull/1580)
 
+* fix: add support for arrays shorter than 2 as arguments for cairo1-run [#1737](https://github.com/lambdaclass/cairo-vm/pull/1737)
+
```
Comment (fmoletta) [inline CHANGELOG.md:156]: We put the latest changes at the top of the list, could you move it?

**Review by fmoletta — APPROVED** [review]

**Review by juanbono — APPROVED** [review]

**Review by pefontana — APPROVED** [review]

**Review by juanbono — CHANGES_REQUESTED** [review]
can you provide an example program where you need to pass less than 2 args?

**Comment by catusax** [comment]
@juanbono tests has been added

**Review by juanbono — APPROVED** [review]

**Pull request merged**